packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/reducer.js

```diff
@@ -197,6 +197,14 @@ const reducer = (state, action) =>
 
         break;
       }
+      case 'CLEAR_RELATION': {
+        const { initialDataPath, modifiedDataPath } = action;
+
+        set(draftState, initialDataPath, []);
+        set(draftState, modifiedDataPath, []);
+
+        break;
+      }
```

Comment:
I know it doesn't add much value at the moment, but could you add a test for this case? 🙏🏼